### PR TITLE
finished e2e test

### PR DESCRIPTION
**Pull request by diegotrasobares**

### CODE REVIEW

File: rebuild.sh [hunk 1]
```diff
@@ -8,7 +8,7 @@ export BINDADDRESS=localhost
 export PORT=8881
 
 function stop_services() {
-  docker-compose down
+  sudo docker compose down
```
Comment (globalworming) [inline rebuild.sh:11]:
```suggestion
  docker compose down
```

File: rebuild.sh [hunk 2]
```diff
@@ -69,7 +69,7 @@ waitForStartup() {
 }
 
 function start_services() {
-  docker-compose up -d --build &
+ sudo docker compose up -d --build &
```
Comment (globalworming) [inline rebuild.sh:72]:
```suggestion
 docker compose up -d --build &
```

File: playwright/models/Steps.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  };
+  clickProceedToCheckoutModal = async () => {
+    await test.step("Click Proceed to checkout btn in the modal", async () => {
+      //Check if the Btn is in the modal or checkout page
+      await this.page.click(
+        `#blockcart-modal > div > div > div.modal-footer > a`
+      );
+    });
+  };
+  fillPersonalInformation = async () => {
```
Comment (diegotrasobares) [inline playwright/models/Steps.js:41]:
```suggestion
  fillPersonalInformation = async (client) => {
```

**Comment by globalworming** [comment]
does not work for me, see https://github.com/headissue/presta-docker-compose/blob/a31ee03c61116cfa243e5c5396379fc0bf54e9a5/tmp/error.txt

i did 
```
bash rebuild.sh
cd playwright
npm run test-local
```
and it seems like no mail is in mailcatcher

**Review by globalworming — CHANGES_REQUESTED** [review]

**Pull request merged**